src/__tests__/Heading.tsx

```diff
@@ -18,7 +19,7 @@ const theme = {
     mono: 'Consolas,monospace'
   },
   lineHeights: {
-    normal: 1.5,
+    default: 1.5,
```

Comment:
this test was using a lineheight that  isn't in the theme spec - typings catch this now :-D